docs/kareen/usecase.md

```diff
@@ -1,89 +1,94 @@
-# TITLE for Use Case Documentation Page
+# CFD Simulations using the Jupyter Notebooks
+**Fei Ding, Ahsan Kareem, Dae Kun Kwon, and Yan Fang**<br/>
+ [NatHaz Modeling Laboratory](https://nathaz.nd.edu/), University of Notre Dame
 
-**Authors, Affiliations**  
+<br/>
 
-Introductory Text.  High-level overview of product. A condimentum vitae sapien pellentesque habitant morbi tristique.
+## Description
 
-## Background 
+The Jupyter Notebooks have been widely used in science and engineering communities as an open-source interactive literate programming paradigm. It has the capabilities of combining text, images or videos with research codes, and can easily document, share, and reproduce data analysis. As for CFD, OpenFOAM is popularly used for computationally establishing wind effects on structures. To help beginners overcome the challenges of steep learning curve posed by OpenFOAM and provide users with the capabilities of generating repetitive jobs and advanced functions, this section presents the work to script the workflow for CFD simulations using OpenFOAM in the Jupyter Notebooks. The developed Jupyter Notebooks can aid in determining inflow conditions, creating mesh files for parameterized building geometries and running the selected solvers. This tool can also contribute to the education for CFD learning as online resources, which will be implemented in the [DesignSafe cyberinfrastructure (CI)](https://www.designsafe-ci.org).
```

Comment:
Could you please remove the Jupyter background info and talk about how to use Jupyter with OpenFOAM.